File: medical-clinic-app/src/main/java/com/medicalclinicapp/medicalclinicapp/models/FisaPatient.java
package com.medicalclinicapp.medicalclinicapp.models;

import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Getter
@Setter
@Table(name = "fisaPatient")
@RequiredArgsConstructor
public class FisaPatient {
    @Id
    @GeneratedValue(strategy= GenerationType.AUTO)
    private Long noFile;

    private String bloodTypePatient;
    private String rhPatient;
    private String allergyPatient;

    //ANAMNEZA

    private String familyHistory;
    private String personalHistory; // ce boli a mai avut, nasteri etc
    private String lifeAndWorkConditional; //daca lucreaza rural sau traieste in zona toxica
    private String behavior;
    private String pillsHistory;

    //EXAMEN OBIECTIV
    private String generalCondition;
    private String waist;
    private String weight;


    //doctorul de familie
    private String presumptiveDiagnostic;

    @OneToOne
    private Patient patient;
}

File: medical-clinic-app-front/src/app/interfaces/fisaPatient.ts
import { Patient } from "./patient";

export class FisaPatient {
    
    bloodTypePatient: string;
    rhPatient: string;
    allergyPatient: string;

    //ANAMNEZA

    familyHistory: string;
    personalHistory: string; // ce boli a mai avut, nasteri etc
    lifeAndWorkConditional: string; //daca lucreaza rural sau traieste in zona toxica
    behavior: string;
    pillsHistory: string;

    //EXAMEN OBIECTIV
    generalCondition: string;
    waist: string;
    weight: string;
    nutritionalStatus: string;
    ganglionSystem: string;
    connectiveTissue: string;

    cardiovascularSystem: string;

    patient: Patient;
}